### PR TITLE
layouts: improve landing page

### PR DESCRIPTION
**Pull request by l-zeuch**
Improve the landing page by using cards instead of a plain list of
links.

We introduce a new partial `"inline/card"`, which we can hook into in
`index.html`. It isn't the most beautiful, but what can one expect of
HTML, really?

**Terms**
- [x] I have read and understood this project's [Contributing Guidelines](CONTRIBUTING.md)

### CODE REVIEW

**Comment by l-zeuch** [comment]
@jo3-l this was your idea — requesting review.

If you're also going to merge this, please do not squash.

File: layouts/partials/card.html [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    <div class="card-body">
+      <div class="d-flex align-items-center mb-3">
+        {{- $colors := slice "yellow" "purple" "green" "red" "blue" }}
+        {{- $color := .color }}
+        {{- if not (in $colors .color) }}
+          {{ $color = "yellow" }}
+        {{- end }}
+        {{ $class := printf "svg-inline-custom mb-0 me-3 card-icon icon-%s" $color }}
+        {{- with .icon }}
+          {{ partial "inline-svg" (dict "src" . "stroke-width" "1.5" "height" "1.875rem" "width" "1.875rem" "class" $class )}}
```
Comment (jo3-l) [inline layouts/partials/card.html:28]:
```suggestion
          {{ partial "inline-svg" (dict "src" . "stroke-width" "1.5" "height" "1.875rem" "width" "1.875rem" "class" $class) }}
```

File: layouts/index.html [hunk 2]
```diff
@@ -24,34 +24,41 @@ <h1>{{ .Title }}</h1>
...
-          <ul>
-            <li><a href="{{ site.BaseURL }}/docs/custom-commands/commands/">Adding custom commands</a></li>
-            <li><a href="{{ site.BaseURL }}/docs/reference/templates/template-scripting/">Template scripting reference</a></li>
-            <li><a href="{{ site.BaseURL }}/docs/reference/templates/functions/">Template functions reference</a></li>
-          </ul>
+          {{ $inner := `<ul>
+<li><a href="{{ site.BaseURL }}/docs/custom-commands/commands/">Adding custom commands</a></li>
+<li><a href="{{ site.BaseURL }}/docs/reference/templates/template-scripting/">Template scripting reference</a></li>
+<li><a href="{{ site.BaseURL }}/docs/reference/templates/functions/">Template functions reference</a></li>
+</ul>`}}
```
Comment (jo3-l) [inline layouts/index.html:31]:
```suggestion
</ul>` }}
```

File: layouts/index.html [hunk 3]
```diff
@@ -24,34 +24,41 @@ <h1>{{ .Title }}</h1>
...
+        {{ $opts := dict "title" "Custom Commands" "icon" "outline/puzzle" "inner" $inner }}
+        {{partial "inline/card.html" $opts}}
```
Comment (jo3-l) [inline layouts/index.html:33]:
```suggestion
        {{ partial "inline/card.html" $opts }}
```

**Review by jo3-l — CHANGES_REQUESTED** [review]
LGTM modulo formatting nits, thanks!

**Comment by l-zeuch** [comment]
Thanks for the review!

**Pull request merged**